edgexfoundry/device-mqtt-go
# refactor!: Use latest SDK for flattened config stem

Signed-off-by: Valina Li <valina.li@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-mqtt-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
v@v-desktop:~/source/repos/device-mqtt-go$ make build
v@v-desktop:~/source/repos/device-mqtt-go$ make test
v@v-desktop:~/source/repos/edgex-compose/compose-builder$ make run no-secty mqtt-broker
v@v-desktop:~/source/repos/device-mqtt-go/cmd$ export EDGEX_SECURITY_SECRET_STORE=false; ./device-mqtt -cp
verify service's config is in Consul at edgex/v3/device-mqtt

## New Dependency Instructions (If applicable)
